src/index/TextScoring.h

```diff
@@ -13,12 +13,13 @@ class ScoreData {
  public:
   ScoreData() = default;
 
-  ScoreData(LocaleManager localeManager) : localeManager_(localeManager){};
+  explicit ScoreData(const LocaleManager& localeManager)
+      : localeManager_(localeManager){};
```

Comment:
The better way to addres the issue here (and for the other occurences below) is to take by value and then move on, so
`ScoreDate(LocManager locManager): locManager_(std::move(locManager)) {}`
(apply for all constructors below).